compiler/rustc_ast/src/ast.rs

```diff
@@ -1688,28 +1688,16 @@ pub enum AttrArgs {
         /// Span of the `=` token.
```

Comment:
So that's true, for example `#[clippy::allow(something)]` is an example of this. However, the path is never resolved, and is instead just string-matched on. In thing that's not possible is `#[custom::whatever(resolve::here)]` (resolving the inner path). However, that's not actually related to what this PR does